### PR TITLE
Zip a Directory

### PR DESCRIPTION
**Pull request by nikhilbarar**
Snippet for `zipDirectory` to zip an entire directory

### CODE REVIEW

**Comment by nikhilbarar** [comment]
Hi @iluwatar I'm done with the changes. Requesting a review.

File: src/main/java/Library.java [hunk 1]
```diff
@@ -312,28 +312,42 @@ public static int httpGet(URL address) throws IOException {
...
-     * @param number number to check prime
-     * @return true if prime
+	
+	/**
+     * Zip an entire directory
+     * @param srcDirName the filename of the source directory
+     * @param zipDirName the filename of the destination zip file
+	 * @throws IOException
      */
-	public static boolean isPrime(int number) {
```
Comment (IAmPramod) [inline src/main/java/Library.java]: I believe this is an unintentional change but please don't remove the existing methods.

File: src/main/java/Library.java [hunk 2]
```diff
@@ -312,28 +312,42 @@ public static int httpGet(URL address) throws IOException {
...
-		// if not, then just check the odds
-		for (int i = 3; i * i <= number; i += 2) {
-			if (number % i == 0) {
-				return false;
-			}
-		}
-		return true;
 	}
+	
+    private static void zipDirectory(File dirToZip, String dirName, ZipOutputStream zipOut) throws IOException {
```
Comment (IAmPramod) [inline src/main/java/Library.java]: I believe we can utilize `zipFiles` method to zip the child files.

**Comment by iluwatar** [comment]
PR closed due to inactivity

**Pull request closed without merging**